services/experimental/pkg/config/defaults/defaultconfig.go

```diff
@@ -0,0 +1,63 @@
+package defaults
+
+import (
+	"github.com/owncloud/ocis/v2/services/experimental/pkg/config"
+	"strings"
+)
+
+func FullDefaultConfig() *config.Config {
```

Comment:
Extra points for adding comments to `FullDefaultConfig` `DefaultConfig` and `EnsureDefaults` :smile: